sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/TimeTravelSpec.scala

```diff
@@ -42,7 +42,7 @@ object TimeTravelSpec {
       }
       val tsToEval = ts.transform {
         case r: RuntimeReplaceable => r.replacement
-        case _: Unevaluable =>
+        case _: Inevaluable =>
```

Comment:
I think `Unvaluable` is fine? I don't think it's worthwhile fixing the whole codebase

Reply:
+1